src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php

```diff
@@ -36,6 +36,7 @@
 
 class EntityTypeTest extends TypeTestCase
 {
+    const ENTITY_TYPE_CLASS = 'Symfony\Bridge\Doctrine\Form\Type\EntityType';
```

Comment:
@nicolas-grekas I agree for the "enhancement" as it considerably eases reading when reviewing and allows autocompletion when writing new tests.

IMHO this should be done in other form tests class too as of 2.8 in a dedicated PR.

Of courses, I can revert it as you wish.

This one is originally made to change alias by fqcn for tests added in #17694.